ntbs-service/wwwroot/css/caseManager.scss

```diff
@@ -0,0 +1,37 @@
+﻿.contact-details-edit-container {
+    display: flex;
+    border-bottom: 1px solid #cccccc;
+    padding-top: 10px;
+    width: 60%;
+}
+
+.contact-details-container {
+    padding-top: 10px;
+    padding-bottom: 5px;
+    width: 60%;
+}
+
+.contact-details-text {
+  margin-top: 10px;
+}
+
+.contact-details-label {
+  margin-top: 10px;
+  font-weight: 800;
+}
+
+.contact-details-fullname {
+  margin-bottom: 10px;
+}
+
+.contact-details-section-heading {
+  margin-bottom: 10px;
+}
```

Comment:
Are we able to use the standard nhs classes for some of these? 
https://service-manual.nhs.uk/design-system/styles/typography#headings?
Also using the `summary-list` (as per the comment in html) might make a couple of these redundant!